modules/bundle/hybrid_programming/scripts/inference.py

```diff
@@ -0,0 +1,51 @@
+# Copyright (c) MONAI Consortium
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import Sequence, Union
+
+import torch
+from monai.bundle import ConfigParser
+from monai.data import decollate_batch
+from monai.utils.enums import CommonKeys
+
+
+def run(config_file: Union[str, Sequence[str]], ckpt_path: str):
+    parser = ConfigParser()
+    parser.read_config(config_file)
+    # edit the config content at runtime and lazy instantiation
+    parser["inferer"]["roi_size"] = [160, 160, 160]
+
+    device = torch.device("cuda:0" if torch.cuda.is_available() else "cpu")
+    # instantialize the components
+    model = parser.get_parsed_content("network").to(device)
+    model.load_state_dict(torch.load(ckpt_path))
+
+    dataloader = parser.get_parsed_content("dataloader")
+    if len(dataloader) == 0:
+        raise ValueError("no data found in the dataloader, please ensure the input image paths are accessable.")
+    inferer = parser.get_parsed_content("inferer")
+    postprocessing = parser.get_parsed_content("postprocessing")
+
+    model.eval()
+    with torch.no_grad():
+        for d in dataloader:
+            images = d[CommonKeys.IMAGE].to(device)
```

Comment:
do you think it'll be more generic if `CommonKeys.IMAGE` is also defined from the config?

Reply:
Sure, updated in the latest commit.

Thanks.